sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_session.py

```diff
@@ -122,12 +121,14 @@ def get_state(self, *, timeout: Optional[float] = None, **kwargs: Any) -> bytes:
         session_state = response.get(MGMT_RESPONSE_SESSION_STATE)  # type: ignore
         return session_state
 
-    def set_state(self, state: Union[str, bytes, bytearray], *, timeout: Optional[float] = None, **kwargs: Any) -> None:
+    def set_state(
+        self, state: Optional[Union[str, bytes, bytearray]], *, timeout: Optional[float] = None, **kwargs: Any
+    ) -> None:
         # pylint: disable=protected-access
         """Set the session state.
 
-        :param state: The state value.
-        :type state: Union[str, bytes, bytearray]
+        :param state: The state value. Setting state to None will clear the current session
```

Comment:
I'm very sorry - baby nit >.<:
```suggestion
        :param state: The state value. Setting state to None will clear the current session.
```